tests/kafkatest/services/kafka/kafka.py

```diff
@@ -354,15 +358,16 @@ def start_cmd(self, node):
 
     def start_node(self, node, timeout_sec=60):
         node.account.mkdirs(KafkaService.PERSISTENT_ROOT)
+
+        self.security_config.setup_node(node)
+        self.security_config.setup_credentials(node, self.path, self.zk_connect_setting(), broker=True)
```

Comment:
Why do we need this change?

Reply:
In the constructor of `SecurityConfig` we don't know node java version.
Therefore we should fix `tls.version` after node version known.
```
    def setup_node(self, node):
...
        if java_version(node) <= 11 and self.properties['tls.version'] == 'TLSv1.3':
            self.properties.update({'tls.version': 'TLSv1.2'})

```